README.md

```diff
@@ -4,6 +4,10 @@
 [![pytest](https://github.com/metno/discovery-metadata-catalog-ingestor/actions/workflows/pytest.yml/badge.svg?branch=main)](https://github.com/metno/discovery-metadata-catalog-ingestor/actions/workflows/pytest.yml)
 [![codecov](https://codecov.io/gh/metno/discovery-metadata-catalog-ingestor/branch/main/graph/badge.svg?token=xSG9Sg0jQ0)](https://codecov.io/gh/metno/discovery-metadata-catalog-ingestor)
 
+## Design
+
+![C4 component diagram](./dmci-component-diagram.png)
```

Comment:
Maybe move it a bit lower in the README?

Reply:
1. Dependencies
2. Environment variables
3. Installation
4. Usage
5. Design
6. Tests
7. Debugging
8. License

OK? Or maybe usage first?

Reply:
Think this is good

Reply:
done